ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh

```diff
@@ -18,7 +18,10 @@ GSHEET_KEY_LOCATION="/ga-gsheet/gcp-sa-account"
 export GSHEET_KEY_LOCATION
 
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking
-pushd e2e-benchmarking/workloads/kube-burner
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+#export EXTRA_FLAGS="--pods-per-node=$PODS_PER_NODE "
+#export NAMESPACED_ITERATIONS=${NAMESPACED_ITERATIONS:-false}
+#export EXTRA_FLAGS=${EXTRA_FLAGS:-}
 export WORKLOAD=node-density-heavy
 
 export CLEANUP_WHEN_FINISH=true
```

Comment:
the cleanup variable for kube-burner ocp way of running is GC, might be nice to add this as an envionrment variable

Reply:
Agreed. We can open a separate PR to tackle that and apply for all relevant workloads.

Reply:
May be Sahil can tackle that as part of having common code across various kube burner workloads.